packages/block-editor/src/components/block-switcher/block-transformations-menu.js

```diff
@@ -49,6 +49,17 @@ function useGroupedTransforms( possibleBlockTransformations ) {
 			},
 			{ priorityTextTransformations: [], restTransformations: [] }
 		);
+		if (
```

Comment:
Is it worth adding an explanatory comment as to why we don't want `core/quote` to be the only priority transform? Just wondered if it'd help anyone looking at this further down the track to know why there's a bit of extra code to shove it out of the way.

Reply:
Ah yes, that's a great point! It should be made clear that if the priority list changes, this code might have to change too.